# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipa/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- IPA-1.01/Global/fft.c.vanilla	2006-08-21 15:33:11.000000000 +0200
+++ IPA-1.01/Global/fft.c	2006-08-21 15:33:21.000000000 +0200
@@ -32,7 +32,7 @@
 #define fail(err)       {failed=true;warn("%s: err",METHOD); goto EXIT;}   
 
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 static void fft_2d(double * data, int nn, int mm, int isign, double * buffer);
 
--- IPA-1.01/Global/Global2.c.vanilla	2006-08-21 15:33:51.000000000 +0200
+++ IPA-1.01/Global/Global2.c	2006-08-21 15:33:56.000000000 +0200
@@ -4,7 +4,7 @@
 #include "Global.h"
 #include "GlobalSupp.h"
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 #define lcUnclassified  0
 #define lcEdge          1
--- IPA-1.01/Global/Global.c.vanilla	2006-08-21 15:33:42.000000000 +0200
+++ IPA-1.01/Global/Global.c	2006-08-21 15:33:45.000000000 +0200
@@ -8,7 +8,7 @@
 
 PImage_vmt CImage;
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 XS( boot_IPA__Global)
 {
--- IPA-1.01/Local/Local.c.vanilla	2006-08-21 15:34:05.000000000 +0200
+++ IPA-1.01/Local/Local.c	2006-08-21 15:34:08.000000000 +0200
@@ -6,7 +6,7 @@
 #include "Local.inc"
 #include "LocalSupp.h"
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 /* Флаги для быстрого Sobel */
 #define SOBEL_COLUMN            0x0001
--- IPA-1.01/Point/Point2.c.vanilla	2006-08-21 15:34:32.000000000 +0200
+++ IPA-1.01/Point/Point2.c	2006-08-21 15:34:36.000000000 +0200
@@ -4,7 +4,7 @@
 #include "Point.h"
 #include "PointSupp.h"
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 #undef METHOD
 #define METHOD "IPA::Point::mask"
--- IPA-1.01/Point/Point.c.vanilla	2006-08-21 15:34:21.000000000 +0200
+++ IPA-1.01/Point/Point.c	2006-08-21 15:34:24.000000000 +0200
@@ -5,7 +5,7 @@
 #include "Point.inc"
 #include "PointSupp.h"
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 #define pix( img, x, y)     ( (img)->type == imByte ? *( (img)->data + (img)->lineSize * (y) + (x)) : ( (img)->type == imShort ? ( ( short*)( (img)->data + (img)->lineSize * (y)))[(x)] : ( ( long*)( (img)->data + (img)->lineSize * (y)))[(x)]));
 
--- IPA-1.01/Geometry/Geometry2.c.vanilla	2006-08-21 15:39:16.000000000 +0200
+++ IPA-1.01/Geometry/Geometry2.c	2006-08-21 15:39:27.000000000 +0200
@@ -4,7 +4,7 @@
 #include "Geometry.h"
 #include "GeometrySupp.h"
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 PImage IPA__Geometry_shift_rotate( PImage img, HV *profile)
 {
--- IPA-1.01/Geometry/Geometry.c.vanilla	2006-08-21 15:39:08.000000000 +0200
+++ IPA-1.01/Geometry/Geometry.c	2006-08-21 15:39:11.000000000 +0200
@@ -5,7 +5,7 @@
 #include "Geometry.inc"
 #include "GeometrySupp.h"
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 PImage_vmt CImage;
 
--- IPA-1.01/Morphology/Morphology.c.vanilla	2006-08-21 15:39:38.000000000 +0200
+++ IPA-1.01/Morphology/Morphology.c	2006-08-21 15:39:47.000000000 +0200
@@ -8,7 +8,7 @@
 #define bwt_pix(x,n)      (((x) ? 1 : 0)<<(n))
 
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 PImage_vmt CImage;
 
--- IPA-1.01/Morphology/Morphology2.c.vanilla	2006-08-21 15:40:04.000000000 +0200
+++ IPA-1.01/Morphology/Morphology2.c	2006-08-21 15:40:08.000000000 +0200
@@ -4,7 +4,7 @@
 #include "Morphology.h"
 #include "MorphologySupp.h"
 
-static SV **temporary_prf_Sv;
+SV **temporary_prf_Sv;
 
 #ifndef max
 #define max(A,B)  ((A) > (B) ? (A) : (B))
